test/e2e/pods.go

```diff
@@ -555,6 +578,233 @@ var _ = Describe("Pods", func() {
 		}, false)
 	})
 
+	It("should reset back-off on image update", func() {
+		name := "backoff"
+		containerName := "back-off"
+		podClient := framework.Client.Pods(framework.Namespace.Name)
+		pod := &api.Pod{
+			ObjectMeta: api.ObjectMeta{
+				Name:   name,
+				Labels: map[string]string{"test": "backoff-image"},
+			},
+			Spec: api.PodSpec{
+				Containers: []api.Container{
+					{
+						Name:    containerName,
+						Image:   "gcr.io/google_containers/busybox",
+						Command: []string{"/bin/sh", "-c", "sleep 1", "/crash/missing"},
+					},
+				},
+			},
+		}
+
+		By("submitting the pod to kubernetes")
+		defer func() {
+			By("deleting the pod")
+			podClient.Delete(pod.Name, api.NewDeleteOptions(0))
+		}()
+		pod, err := podClient.Create(pod)
+		if err != nil {
+			Failf("Failed to create pod: %v", err)
+		}
+
+		expectNoError(framework.WaitForPodRunning(pod.Name))
+
+		By("verifying the pod is in kubernetes")
+		pod, err = podClient.Get(name)
+		if err != nil {
+			Failf("failed to get pod: %v", err)
+		}
+
+		By("Build backoff offset")
```

Comment:
Please be consistent with grammar and capitalization.  i.e. "building backoff offset".